### PR TITLE
Remove default adapters

### PR DESCRIPTION
**Pull request by lherman-cs**
* Update examples to import adapters explicitly
* Remove default adapters to be imported implicitly

### CODE REVIEW

File: examples/screenshare/main.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ import (
 	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
 	_ "github.com/pion/mediadevices/pkg/codec/opus"     // This is required to register opus audio encoder
 	_ "github.com/pion/mediadevices/pkg/codec/vpx"
+	_ "github.com/pion/mediadevices/pkg/driver/camera"     // This is required to register camera adapter
+	_ "github.com/pion/mediadevices/pkg/driver/microphone" // This is required to register microphone adapter
```
Comment (at-wat) [inline examples/screenshare/main.go]: This may be omitted in screenshare example.

File: examples/rtp-send/main.go [hunk 2]
```diff
@@ -6,8 +6,10 @@ import (
 	"os"
 
 	"github.com/pion/mediadevices"
-	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
-	_ "github.com/pion/mediadevices/pkg/codec/vpx"      // This is required to register VP8/VP9 video encoder
+	_ "github.com/pion/mediadevices/pkg/codec/openh264"    // This is required to register h264 video encoder
+	_ "github.com/pion/mediadevices/pkg/codec/vpx"         // This is required to register VP8/VP9 video encoder
+	_ "github.com/pion/mediadevices/pkg/driver/camera"     // This is required to register camera adapter
+	_ "github.com/pion/mediadevices/pkg/driver/microphone" // This is required to register microphone adapter
```
Comment (at-wat) [inline examples/rtp-send/main.go]: Audio is not used in this example. It can be omitted.

File: examples/screenshare/main.go [hunk 3]
```diff
@@ -8,6 +8,8 @@ import (
 	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
 	_ "github.com/pion/mediadevices/pkg/codec/opus"     // This is required to register opus audio encoder
 	_ "github.com/pion/mediadevices/pkg/codec/vpx"
```
Comment (at-wat) [inline examples/screenshare/main.go]: Sorry, I had forgotten to add a comment to vpx import. Would you add it?

File: examples/screenshare/main.go (hunk 1 above)
Comment (lherman-cs) [reply examples/screenshare/main.go]: Ah, right

File: examples/rtp-send/main.go (hunk 2 above)
Comment (lherman-cs) [reply examples/rtp-send/main.go]: my bad, copy-paste error lol

File: examples/screenshare/main.go (hunk 3 above)
Comment (lherman-cs) [reply examples/screenshare/main.go]: Yep, I'll add it

**Comment by lherman-cs** [comment]
Updated the PR.

**Pull request merged**